server/.clj-kondo/hooks/defsql.clj

```diff
@@ -0,0 +1,22 @@
+(ns hooks.defsql
+  (:require [clj-kondo.hooks-api :as api]))
+
+
+(defn defsql [{:keys [node]}]
```

Comment:
Neat! Curious, what made you want to add this in?

Reply:
There's a lot of boilerplate and I'm adding more with the graceful query shutdown.